xbmc/network/WebServer.cpp

```diff
@@ -102,9 +102,25 @@ CWebServer::CWebServer()
 #endif
 }
 
+static MHD_Response* create_response(size_t size, void* data, int free, int copy)
```

Comment:
I would prefer having an enum here instead of free and copy as it would make it clearer at the call sites what's going on instead of mostly passing `MDH_NO, MHD_NO`.
I realize this is a bit about personal taste and probably out of scope for this PR but I thought I'd mention it